modern/gcc/Dockerfile

```diff
@@ -67,3 +67,6 @@ RUN sudo rm -rf /usr/lib/gcc/x86_64-linux-gnu/* \
     && sudo ldconfig -C /etc/ld.so.cache \
     && conan profile new --detect --force default \
     && conan profile update settings.compiler.libcxx=libstdc++11 default
+
+RUN conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock binutils/2.37@ --build=missing \
+    && rm -rf /home/conan/.conan/data
```

Comment:
```suggestion
RUN conan install -r conancenter -g deploy -if /opt/conan -l /opt/conan/conan.lock binutils/2.37@ \
    && rm -rf /home/conan/.conan/data
```

Reply:
And that's the problem. We don't have that package available for gcc11. Which make me think the fragility using Conan to this task. I'm considering building from source.

Reply:
Do we need the one for gcc11? It should be just an executable... In fact, if we consider it a _tool_, the `compiler` setting should be removed.